### PR TITLE
Added Slow TF2T

### PR DESCRIPTION
**Pull request by ranjinidas**
Corrected strategy from last pull request.

### CODE REVIEW

File: axelrod/strategies/titfortat.py [hunk 1]
```diff
@@ -386,3 +386,33 @@ def reset(self):
...
+
+class SlowTitForTwoTats(Player):
+    """
+    A player that plays [c,c], then if the opponent plays the same move twice, 
+    plays that move 
+    """
+
+    name = 'Slow Tit For Two Tats'
+    classifier = {
+        'memory_depth': 2,  # Four-Vector = (1.,0.,1.,0.)
```
Comment (drvinceknight) [inline axelrod/strategies/titfortat.py]: The `# Four-Vector = (1., 0., 1., 0.)` isn't needed anymore.

**Comment by drvinceknight** [comment]
This is looking good @ranjinidas! Thanks!

We're still missing a couple of tests, we'd be happy to help:

1. Would you like to do this yourself?
2. Would you like one of us to walk you through it?
3. Would you like one of us to do it?

Whatever you would prefer :)

Also, as a side note. I think (although I'm not sure) the errors you were having might have been because the doctests failed half way through and there were some files written that were not deleted... That's just a hunch though: they all passed fine on travis (a system hooked in to github that runs the test suite for every PR).

File: axelrod/strategies/titfortat.py [hunk 2]
```diff
@@ -386,3 +386,33 @@ def reset(self):
         Player.reset(self)
         self.contrite = False
         self._recorded_history = []
+
+
+class SlowTitForTwoTats(Player):
+    """
+    A player that plays [c,c], then if the opponent plays the same move twice,
```
Comment (drvinceknight) [inline axelrod/strategies/titfortat.py]: Could we write "A player that plays C twice, ...". This is not terribly important: just looks nicer to me :)

**Comment by ranjinidas** [comment]
Hi @drvinceknight, thanks for reviewing this! I have a test written but can check with you or someone else on Gitter before adding it in.

**Comment by drvinceknight** [comment]
Awesome: I'm online now if that's helpful :)

File: axelrod/strategies/titfortat.py [hunk 3]
```diff
@@ -386,3 +386,33 @@ def reset(self):
...
+        'stochastic': False,
+        'makes_use_of': set(),
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def strategy(self, opponent):
+        #Play [c, c]
+        if len(self.history)<2:
```
Comment (drvinceknight) [inline axelrod/strategies/titfortat.py]: Tiny bit of [PEP8](https://www.python.org/dev/peps/pep-0008/):

```
if len(self.history) < 2:
```

**Comment by drvinceknight** [comment]
This looks great to me! 👍 

I've left one very minor PEP8 comment. We have a two core dev review policy so once one of the others has taken a look (and possibly added their own comments etc) this will go in :) Thanks a lot for working on this.

File: axelrod/strategies/titfortat.py (hunk 3 above)
Comment (ranjinidas) [inline axelrod/strategies/titfortat.py]: Added this in! Thanks!

File: axelrod/strategies/titfortat.py [hunk 4]
```diff
@@ -386,3 +386,34 @@ def reset(self):
...
+        'stochastic': False,
+        'makes_use_of': set(),
+        'inspects_source': False,
+        'manipulates_source': False,
+        'manipulates_state': False
+    }
+
+    def strategy(self, opponent):
+        
+        #Play [c, c]
```
Comment (marcharper) [inline axelrod/strategies/titfortat.py]: # Start with two cooperations

**Comment by marcharper** [comment]
Looks good to me, we can sweep up any PEP8 issues sometime later.

**Pull request merged**